Tentacle/Client.swift

```diff
@@ -88,6 +99,14 @@ extension URLSession {
 	}
 }
 
+public enum HTTPMethod: String {
```

Comment:
Does this need to be `public`? Let's make it `private` or `internal` if possible.